### DISCUSSION TITLE
是否支持双链？

### DISCUSSION THREAD

**Discussion by Mojicode** [toeverything/AFFiNE#2007] [General]
如题，没有找到如何使用双链引用

**Comment by Mojicode** [comment]
help, 不知道怎么操作啊，块上右键没反应，反斜杠也没有对应的命令，只能使用@链接页面吗，块级的双向链接没有吗

**Reply by doodlewind** [reply]
目前双链有 `@` 和 `[[` 两个入口，但还属于上线前的调试阶段，因此细节打磨得还不够。我们计划在下一个大版本中引入块级双链（具体命名和交互方式待定）。感谢反馈！

**Comment by HeJiachen-PM** [comment]
Hello。双链支持已经上线了。

Yifeng Wang ***@***.***> 于 2023年4月21日周五 10:04写道：

**Reply by Mojicode** [reply]
是支持块级双链了吗？

**Reply by houjoe0829** [reply]
块级双链还没有支持，还在进行中。
